docs/DeveloperGuide.md

```diff
@@ -1125,24 +1182,56 @@ testers are expected to do more *exploratory* testing.
     1. Prerequisites: User is currently viewing the task list page. Only the following 3 tasks are be added to NurseyBook. We shall call them Tasks A, B and C.
         * Task A: A non-recurring task, the date of which is current date. <br> Assume this date is `2021-11-12` for illustration purposes.
         * Task B: A daily recurring task, the initial date of which is current date + 2 days. <br> Assume this date is `2021-11-14` for illustration purposes.
-        * Task C: A non-recurring task, the date of which is current date + 4 days. <br> Assume this date is `2021-11-16` for illustration purposes. 
+        * Task C: A non-recurring task, the date of which is current date + 4 days. <br> Assume this date is `2021-11-16` for illustration purposes.
     2. Test case - today:`viewSchedule 2021-11-12` <br>
        Expected: Task A should be displayed.
-   
+
     3. Test case - one day ahead:`viewSchedule 2021-11-13` <br>
        Expected: No tasks should be displayed.
-   
+
     4. Test case - four days ahead:`viewSchedule 2021-11-16` <br>
        Expected: Tasks B and C should be displayed. Date of Task B in this display should be `2021-11-16`.
-   
+
     5. Test case - a week ahead:`viewSchedule 2021-11-19` <br>
        Expected: Task B should be displayed. Date of Task B in this display should be `2021-11-19`.
-   
-    6. Some other incorrect `viewSchedule` commands to try (error details shown in status message): 
+
+    6. Invalid commands to try (Error details shown in the status message):
         * Date input has already passed: `viewSchedule 2021-11-11`
         * Not formatting the date correctly in yyyy-mm-dd format: `viewSchedule 16-12-2021`
```

Comment:
Maybe also dates that go beyond the acceptable 12 weeks?

Reply:
Ok will add it in!